ios/Sources/Styleguide/Resources/ja.lproj/Localizable.strings

```diff
@@ -0,0 +1,13 @@
+// MARK: Utility
+
```

Comment:
I think these lines are not necessary! 
Please remove this 🙏🏼

Reply:
Thanks for review! I just removed 👍